### PR TITLE
Github Deployments: Update some copies to be clearer

### PR DESCRIPTION
**Pull request by vykes-mac**
Related to #

## Proposed Changes
This PR is based off feedback from @paulopmt1 developer interview pet6gk-YL-p2

* on the repository empty page change copy to 'Get started by adding a GitHub account.' In the previous copy "GitHub installation" wasn't clear to the user
* Change repository list CTA to "Connect"

## Testing Instructions
**Before**
image
image

**After**
image

image

* 

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-aUh-p2)?

### CODE REVIEW

File: client/my-sites/github-deployments/components/repositories/browse-repositories.tsx [hunk 1]
```diff
@@ -47,7 +47,7 @@ export const GitHubBrowseRepositories = ( {
 		if ( ! installation ) {
 			return (
 				<Card css={ { display: 'flex', flexDirection: 'column', alignItems: 'center', margin: 0 } }>
-					<span>{ __( 'Add a GitHub installation to import an existing repository.' ) }</span>
+					<span>{ __( 'Get started by adding a GitHub account.' ) }</span>
```
Comment (zaguiini) [inline client/my-sites/github-deployments/components/repositories/browse-repositories.tsx:50]: I don't think we should change this one. The CTA says "Add GitHub account," and we must be consistent and point to things with similar copies.

Comment (zaguiini) [reply client/my-sites/github-deployments/components/repositories/browse-repositories.tsx:50]: I can see the 'why' of this change now. And sorry, I misread that. I thought we were saying "Add installation" in the CTA but it says account.

Comment (zaguiini) [reply client/my-sites/github-deployments/components/repositories/browse-repositories.tsx:50]: I was gonna say that we should mention "create a repository" but that's going to be dropped.

**Review by zaguiini — APPROVED** [review]

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/11572484

Some locales (Hebrew, Japanese) have been temporarily machine-translated due to translator availability. All other translations are usually ready within a few days. Untranslated and machine-translated strings will be sent for translation next Monday and are expected to be completed by the following Friday.

Thank you @vykes-mac for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/11572484).

**Pull request merged**